packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -498,7 +498,7 @@ export async function reconcileGoCardlessTransactions(acctId, transactions) {
 
       // Update the transaction
       const updates = {
-        date: trans.date,
+        date: existing.date,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I think we can remove this field entirely. Then it won't perform the UPDATE operation on it thus making it immutable.

```suggestion
```

Reply:
Ah indeed, tested and it works.